content/docs/setup/kubernetes/multicluster-install/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Multicluster Installation
```

Comment:
I am merely cleaning up what exists already. And I don't want to take up fancy buzzwords like Multi-cloud. I would much prefer to stick to hard facts, which is that we are spanning a mesh across multiple kubernetes clusters.. whether they are on same or different cloud is irrelevant

Reply:
The "fancy buzzword" was defined by the environments working group to represent the technologies as there is more than one that enable a mesh across multiple kubernetes clusters.  In the case of mesh expansion, multicloud is the correct terminology and the overall heading of "Multicluster" makes no sense in that context.

Reply:
@sdake I don't really think multicloud is a good name for mesh expansion. It's really more of a cluster expansion, isn't it? I tend to like the way @rshriram has described the old multicluster and the zvpn as two different configurations for multicluster. They are both really just an Istio mesh spanning multiple clusters, with different runtime configurations for doing it. On the other hand, mesh expansion seems like a very different thing so putting all three under the same multicloud folder would seem strange to me.

Reply:
and in case of mesh expansion, federation is more fitting than multicloud. There is nothing here that says everything should be on different clouds, or anything that is specific to connecting things in multiple clouds that is different from connecting things on the same cloud.

Reply:
Let's keep the updates in this PR with the current name - and move the 'multicluster vs federation' to a different topic/PR. 

The Multicluster we released in 1.0 will need to document the new SNI-routing and change the 'limitations' sections  - since we can now support additional features.

Reply:
@costinm is right. The change from multicluster to multi-cloud needs to happen on places beyond the scope of this PR and would be better done on their own effort. For consistency though, consider keeping the verb at the beginning of the title: 
Installing Multiclusters